src/routes/StandaloneRouting.js

```diff
@@ -1,104 +1,114 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-import qs from 'query-string'
-import Viewer from "../connectedComponents/Viewer";
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import qs from 'querystring';
```

Comment:
Interesting:

- [`querystring`](https://www.npmjs.com/package/querystring) has not been updated in 6-years
- [`query-string`](https://www.npmjs.com/package/query-string) has been updated within the last month, and appears to be commonly bundled with `create-react-app`

We may want to double check why we were having issues with `query-string`, as it appears to be the package we want.

Reply:
It looks like `query-string` package is missing. `querystring` package is probably added as a dependency of another package.

I think we can add `query-string` package and continue to use it here since we use it explicitly.

Reply:
I rolled it back. Thanks @evren217